onnxruntime/core/optimizer/layer_norm_fusion.cc

```diff
@@ -345,6 +345,18 @@ X --> Pow --> ReduceMean --> Add --> Sqrt --> Div --> Mul
 |                                              ^
 |                                              |
 +----------------------------------------------+
+Additional FP16 patterns supported if allow_precision_change_ is true:
+X --> Cast1 --> Pow --> ReduceMean --> Add --> Sqrt --> Div --> Cast2 --> Mul
+        |                                               ^                  ^
+        |                                               |                  |
+        |-----------------------------------------------+                Scale
```

Comment:
>| [](http://example.com/codeflow?start=8&length=1)

nit: use + ,  same for below, also, would be more readable with one line before/after the graph